api/src/wfl/api/routes.clj

```diff
@@ -45,8 +47,8 @@
            :swagger   {:tags ["Informational"]}}}]
    ["/api/v1/append_to_aou"
     {:post {:summary    "Append to an existing AOU workload."
-            :parameters {:body ::spec/append-to-aou-request}
-            :responses  {200 {:body ::spec/append-to-aou-response}}
+            :parameters {:body ::aou/append-to-aou-request}
+            :responses  {200 {:body ::aou/append-to-aou-response}}
```

Comment:
I was a little bit concerned if the spec names will potentially conflict with function names within their namespaces, looks like we don't have conflicts here!